Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+aiohttp = "*"
+bitshares = "*"
+sqlalchemy = "*"
+psycopg2 = "*"
+aiopg = "*"
+alembic = "*"
+pyyaml = "*"
+marshmallow = "*"
+gunicorn = "*"
+pytest = "*"
+pytest-asyncio = "*"
+pytest-aiohttp = "*"
+
+[requires]
+python_version = "3.7"
```

Comment:
Use semantic versioning so that when you update Pipfile.lock, dependencies compatible with the current code base are install.